src/styles/sections/header.scss

```diff
@@ -14,10 +14,38 @@
     .header__links {
         position: relative;
 
+        .header__link {
+            margin: 0 10px;
+            vertical-align: top;
+            position: relative;
+            top: 8px;
+        }
+
         img {
             height: 35px;
-            margin: 10px;
             width: auto;
+            position: relative;
+        }
+
+        .header__link:nth-child(2) {
```

Comment:
@ArtOfCode- @ranolfi @luap42 RE coding standards - Pretty sure that's not BEM. I vaguely remember @ArtOfCode- mentioning he needs to change them. 

BEM follows the {COMPONENT_NAME}__{SUB_COMPONENT}--{MODIFIER}, unless you're referencing something else im not seeing @luap42 ?

We should be following standard BEM with additional atomic modifiers rather than a slightly modified version.

Reply:
https://forum.codidact.org/t/what-style-should-we-use-for-css-naming/486

It appears, that most people opposed that change. (the 3 likes on Art's post are by those, who then write a message of disagreement)

Reply:
Aye, @mattjbrent, we're not quite using BEM - it's a bit of a mashup of BEM and atomics and predicates... I'm not thrilled about it, but that's what folks thought was best, so that's what we're working on.

Reply:
Strong advocate of practical `is-something` classes. Order is something to be used when you want a list with different odd/even items (that being said, that is my opinion). I'm also strongly advocating for `js-something` classes, but that's mainly what I am used to.

Reply:
@ArtOfCode-  @ranolfi  I'm totally for modifier classes. But im not sure that using a very close but slightly different version of a widely used practice is the best approach here for class names.

I agree with everything else - just our approach with naming classes. If we are going to use something else im fine with that but i think using `name--element__modifier` is far too close to `name__element--modifer` ... seems unnecessarily confusing to me?

Reply:
@mattjbrent I think there are two separate concerns in this thread. One is the use of BEM vs. BEM-but-with-modifiers (which we're using the latter of, i.e. `.{COMPONENT}__{SUB_COMPONENT}.is-{MODIFIER}`). The other is the use of `:nth-child` to style the project links, which is what @luap42 was pointing out, which would be better done with something else - probably an ID on each link instead, so that if we change the order we don't have to change the CSS. The latter of those two issues is the one that @ranolfi has requested changes for before merging.

Reply:
Indeed, and I have now approved the changes. I can't comment on the naming scheme right now as I've yet to review the definitions. @ArtOfCode- please merge at will.